.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,16 +1,18 @@
 ---
-name: Bug Report
-about: Create a report to help us improve
+name: Bug Report(.Net SDK)
+about: Create a report to help us improve AI .NET SDK
 labels: bug
 ---
 
-**Describe your environment.** Describe any aspect of your environment relevant to the problem:
+**Is this a bug in .NET SDK?**
+             If yes, proceed to Bug Template. 
+             If not, please file it under the appropriate AI SDK repro path: [Javascript SDK](https://github.com/Microsoft/ApplicationInsights-js/issues) ; [Java SDK](https://github.com/Microsoft/ApplicationInsights-Java/issues) ; [Node.JS SDK](https://github.com/Microsoft/ApplicationInsights-node.js/issues)
```

Comment:
I am not sure if we need to link to other language SDKs from here. We never had anyone reporting a node.js issue in this repo.

Azure functions is a good candidate to be put here.

Reply:
Sure, I'll add Azure functions here as well. But I want this message to be consistent across all templates, hence JS is added over here.